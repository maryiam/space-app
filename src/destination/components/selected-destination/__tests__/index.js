import renderer from 'react-test-renderer';
import React from 'react';
import SelectedDestination from '../index';

describe('selected-destination/', () => {
  describe('#render', () => {
    it('renders correctly with default props', () => {
      const tree = renderer.create(<SelectedDestination />);
      expect(tree.toJSON()).toMatchSnapshot();
    });

    it('renders correctly with selected destination data', () => {
      const tree = renderer.create(
        <SelectedDestination
          name="toto"
          climate="tropical"
          gravity="1 standard"
          img="/assets/planets/thumb/yavin4@x2.png"
          population="100"
          price={45}
        />
      );
      expect(tree.toJSON()).toMatchSnapshot();
    });
  });
});
