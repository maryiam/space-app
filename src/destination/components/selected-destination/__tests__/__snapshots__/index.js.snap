// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`selected-destination/ #render renders correctly with default props 1`] = `
<div
  className="sc-bdVaJa iHKLJG"
>
  <img
    className="sc-bwzfXH bXuwR"
    src={null}
  />
  <div
    className="sc-htpNat eJgCND"
  >
    <h2
      className="sc-bxivhb bUZGRr"
    >
      unknown
    </h2>
    <span
      className="sc-ifAKCX cFlEyZ"
    >
      Climate: 
      N/A
    </span>
    <span
      className="sc-ifAKCX cFlEyZ"
    >
      Gravity: 
      N/A
    </span>
    <span
      className="sc-ifAKCX cFlEyZ"
    >
      Population: 
      N/A
    </span>
    <span
      className="sc-ifAKCX cFlEyZ"
    >
      Price: 
      N/A
    </span>
  </div>
</div>
`;

exports[`selected-destination/ #render renders correctly with selected destination data 1`] = `
<div
  className="sc-bdVaJa iHKLJG"
>
  <img
    className="sc-bwzfXH bXuwR"
    src="/assets/planets/thumb/yavin4@x2.png"
  />
  <div
    className="sc-htpNat eJgCND"
  >
    <h2
      className="sc-bxivhb bUZGRr"
    >
      toto
    </h2>
    <span
      className="sc-ifAKCX cFlEyZ"
    >
      Climate: 
      tropical
    </span>
    <span
      className="sc-ifAKCX cFlEyZ"
    >
      Gravity: 
      1 standard
    </span>
    <span
      className="sc-ifAKCX cFlEyZ"
    >
      Population: 
      100
    </span>
    <span
      className="sc-ifAKCX cFlEyZ"
    >
      Price: 
      45
    </span>
  </div>
</div>
`;
